### PR TITLE
[rt-thread] add rt-thread os in readme

### PR DESCRIPTION
**Pull request by mysterywolf**
**Describe the PR**
A clear and concise description of what this PR solve.

**Additional context**
If applicable, add any other context about the PR and/or screenshots here.

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -90,6 +90,7 @@ TinyUSB is completely thread-safe by pushing all Interrupt Service Request (ISR)
 
 - **No OS**
 - **FreeRTOS**
+- **RT-Thread** (https://github.com/tfx2001/tinyusb)
```
Comment (hathach) [inline README.rst:93]: it should be linked to `https://github.com/RT-Thread/rt-thread`, though why would you close the PR ?

Comment (mysterywolf) [reply README.rst:93]: there is a problem on my side. when i fix that, i will re create a pr.

**Comment by mysterywolf** [comment]
This PR is ready to review

**Review by hathach — APPROVED** [review]
thank you, I pushed an update to correct link format in rst file. Will merge when ci passes

**Pull request merged**